src/Xamarin.Android.Build.Tasks/Tasks/AsyncTask.cs

```diff
@@ -261,12 +282,26 @@ private void LogWarnings ()
 			}
 		}
 
+		private void LogCustomData ()
+		{
+			lock (customMessageQueue.SyncRoot) {
```

Comment:
Ditto this body: `LogCustomData()` looks like `LogErrors()` looks like `LogWarnings()`... Surely we can have a helper method for consistency.